app/views/kaminari/_paginator.html.erb

```diff
@@ -0,0 +1,17 @@
+<%= paginator.render do -%>
+  <nav class="pagination" role="navigation" aria-label="pager">
+    <%= first_page_tag unless current_page.first? %>
+    <%= prev_page_tag unless current_page.first? %>
```

Comment:
A suggestion to move the `unless current_page.first?` to its own line. I.e.:
```
<% unless current_page.first? -%>
  <%= first_page_tag %>
   <%= prev_page_tag %>
<% end %>
```

Reply:
Done in sha `8f2b2bf`